### ISSUE TITLE
Price slider faceted not working correct with Multistore

### ISSUE THREAD

**Issue by 01park** [PrestaShop/PrestaShop#17900]
Im having an issue with sorting price from low to high - high to low with multiple - multishops. I noticed, the shops that have the issue, do not have the price slider from faceted. 

I uninstalled Faceted and installed it with the same version ( latest one ). Rebuild attributes, cleared cache etc. No matter what I do, the price slider will not appear on most shops. I dont know why it shows on some and on orthers nothing. 

When I check the error-log. the issue is also showing.

ps_facetedsearch/src/Product/SearchProvider.php(137): PrestaShop\PrestaShop\Core\Product\Search\ProductSearchResult->setProducts(false)
#1 classes/controller/ProductListingFrontController.php(308): PrestaShop\Module\FacetedSearch\Product\SearchProvider->runQuery(Object(Context), Object(PrestaShop\PrestaShop\Core\Product\Search\ProductSearchQuery))
#2 
classes/controller/ProductListingFrontController.php(580): ProductListingFrontControllerCore->getProductSearchVariables()
#3 /home/pa in 
src/Core/Product/Search/ProductSearchResult.php on line 61
[29-Feb-2020 15:26:04 Europe/Dublin] PHP Fatal error:  Uncaught TypeError: Argument 1 passed to PrestaShop\PrestaShop\Core\Product\Search\ProductSearchResult::setProducts() must be of the type array, boolean given, called in 
ps_facetedsearch/src/Product/SearchProvider.php on line 137 and defined in src/Core/Product/Search/ProductSearchResult.php:61

Prestashop 1.7.6.3, but already had this issue from 1.7.5.2

**Comment by khouloudbelguith** [comment]
Hi @01park,

I did not manage to reproduce the issue with
1. PS1.7.5.2 and ps_facetedsearch v2.2.0.
2. PS1.7.5.2 and ps_facetedsearch v3.4.1.
3. Ps1.7.6.0 and ps_facetedsearch v3.4.0.
4. PS1.7.6.0 and ps_facetedsearch v3.4.1.
5. PS1.7.6.3 and ps_facetedsearch v3.4.1.

I attached a screen record
https://drive.google.com/file/d/1EdSCVIIW9Z1qQIK4NPJs3Ml7MlB6ZcMy/view

Thanks to check & feedback.
